environment/prod/deployment/beta/docker-compose.beta.yml

```diff
@@ -59,6 +59,7 @@ services:
       timeout: 3s
       retries: 5
     volumes:
+      - ./environment/prod/deployment/postgres/init-unaccent.sql:/docker-entrypoint-initdb.d/init-unaccent.sql
```

Comment:
```Warning: scripts in /docker-entrypoint-initdb.d are only run if you start the container with a data directory that is empty;
```
source: https://hub.docker.com/_/postgres

So it will only work on fresh database. If database has been already created you need to manually remove current volumes and deploy fresh app.

Reply:
Ok, I understand. At the moment, it's hard to say whether we will ever have to freshly implement the app, but maybe there will be such a need in the future.